src/interfaces/IPRBProxyRegistry.sol

```diff
@@ -246,7 +246,7 @@ interface IPRBProxyRegistry {
     /// Requirements:
     /// - The caller must have a proxy.
     ///
-    /// @param envoy The address of the account being given or taken permission to call the target.
+    /// @param envoy The address of the account from which permission is given or taken to call the target.
```

Comment:
This still doesn't look right, grammatically-wise...

Reply:
This is what ChatGPT recommended in the screenshot you shared in the other PR comment thread

Reply:
So I'll keep it like this, it's fine

Reply:
As I've said in that, other thread, ChatGPT's alternative wasn't good either.

Just read the first part of the sentence: "The address of the account from which permission is given..." Does "given from" sound right to you?

Reply:
I mean... That would, indeed, sound right, but in a different context: when it's the envoy "from whom" the permission is given. Here, it's "given to" him. Or "taken from" him, in the other case.

Reply:
Though, don't misinterpret this discussion as me trying to cause you trouble before the release. 😄  

I'm just making a point (which, I think, is justified) - and it's up to you whether to implement this or not.

Reply:
Nah, you're right .. I'll polish the wording directly on `main`.